test/services/dumper/dumper-sequelize.test.js

```diff
@@ -53,6 +55,22 @@ describe('services > dumper > sequelize', () => {
     cleanOutput();
   });
 
+  it('should generate a model file with belongsTo associations and sourceKey/targetKey', async () => {
+    expect.assertions(2);
+    const dumper = getDumper();
+    await dumper.dump(simpleModelNonPrimary);
+    await dumper.dump(belongsToModelNonPrimary);
```

Comment:
You can avoid the double usage of `dump` function by
```suggestion
    await dumper.dump({ ...simpleModelNonPrimary, ...belongsToModelNonPrimary });
```